ROX#21157: update installation docs for init bundle changes and other issues (#71919)
openshift/openshift-docs: modules/create-resource-init-bundle.adoc

:_mod-docs-content-type: PROCEDURE
[id="create-resource-init-bundle_{context}"]
= Applying the init bundle on the secured cluster

//Do not show for ACSCS
ifndef::cloud[]

> Before you install secured clusters, you must apply the init bundle by using it to create the required resources on the cluster that will allow the services on the secured clusters to communicate with Central.

Reviewer: Given the heading **on the secured cluster** here are questions as a reader:
1. Before you install **secured clusters** (plural) means install **secured cluster services**?
2. required resources on **the** cluster: this is the to-be-secured cluster?
3. the services on the **secured clusters** (plural) would singular make it clearer it is same cluster as item 2?

Author: Rewriting to avoid the awkwardness, making cluster singular was a good suggestion!

.Prerequisites
* You must have generated an init bundle containing secrets.

> * You must have created the `stackrox` project, or namespace, on the cluster where Central is installed. Using `stackrox` for the project is not required, but ensures that false positives for {product-title-short} processes are not reported when scanning your clusters.

Reviewer: Is this prerequisite only for self-managed? I am not sure whether this is within a conditional block, as above.

If I understand correctly, cloud service customers do not have access to the cluster where Central is installed.

Author: This should actually read *...on the cluster where the secured cluster services will be installed.* So it applies to both.

Reviewer: Aha, that makes sense!

.Procedure
//Show for OpenShift
ifdef::openshift[]
To create resources, perform only one of the following steps:

> * Create resources using the {ocp} web console: In the {ocp} web console, make sure that you are in the `stackrox` namespace. In the top menu, click *+* to open the *Import YAML* page. You can drag the init bundle file or copy and paste its contents into the editor, and then click *Create*. If you successfully completed this step, the display shows that the `collector-tls`, `sensor-tls`, and admission-control-tls` resources were created.

Reviewer 2: To me, _If you successfully completed this step_ sounds like a challenge.
```suggestion
* Create resources using the {ocp} web console: In the {ocp} web console, make sure that you are in the `stackrox` namespace. In the top menu, click *+* to open the *Import YAML* page. You can drag the init bundle file or copy and paste its contents into the editor, and then click *Create*. When the command is complete, the display shows that the `collector-tls`, `sensor-tls`, and admission-control-tls` resources were created.
```

Author: It was a challenge for me :D but I'll reword....

<1> Create the project where secured cluster resources will be installed. This example uses `stackrox`.
<2> Specify the file name of the init bundle containing the secrets.
<3> Specify the project name that you created. This example uses `stackrox`.
endif::kube[]

.Next Step
* Install {product-title-short} secured cluster services in all clusters that you want to monitor.

ifeval::["{context}" == "init-bundle-cloud-ocp"]

> :!openshift:

Reviewer 3: I don't really understand the syntax but could you please explain why this is with exclamation mark?
I have noticed that sometimes such blocks are the same as in the begining (e.g. https://github.com/openshift/openshift-docs/blob/4c928284d0f9c9cc2919d2e96d7bcaf3b11e4c1d/modules/deploy-app.adoc?plain=1#L11 )
